apps/src/sites/studio/pages/lessons/show.js

```diff
@@ -138,6 +141,15 @@ function prepareExpandableImageDialog() {
   );
 }
 
+function displayDifferentiationChat() {
+  const differFabMountPoint = document.getElementById(
+    'differentiation-fab-mount-point'
```

Comment:
recommendation: standardize on spelling, per earlier comment

Reply:
Do you think just calling it "AiDiff" is fine? Differentiation is such a long word that I never spell right on the first try

Reply:
yes, I think AiDiff is fine. `Aidiff` is also an option with some precedent, see Ailab and Aichat

Reply:
the reason student labs use those spellings is so that they can do a case-insensitive search for "ailab" and not miss things spelled in snake or kebab case. I think that makes sense for something like a lab name since the config for each lab gets super spread out, but might be overkill for this case. you can decide, I just vote for consistency